seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect/JdbcDialect.java

```diff
@@ -438,72 +441,165 @@ default String convertType(String columnName, String columnType) {
     /**
      * Refresh physical table schema by schema change event
      *
-     * @param sourceDialectName source dialect name
      * @param event schema change event
-     * @param jdbcConnectionProvider jdbc connection provider
-     * @param sinkTablePath sink table path
+     * @param connection jdbc connection
+     * @param tablePath sink table path
      */
-    default void refreshTableSchemaBySchemaChangeEvent(
-            String sourceDialectName,
-            AlterTableColumnEvent event,
-            JdbcConnectionProvider jdbcConnectionProvider,
-            TablePath sinkTablePath) {}
+    default void applySchemaChange(
+            SchemaChangeEvent event, Connection connection, TablePath tablePath)
+            throws SQLException {
+        if (event instanceof AlterTableColumnsEvent) {
```

Comment:
cc @dailai . Looks like add jdbc supported schema evolution? Not only jdbc mysql.

Reply:
Only compatible with MySQL database, other databases need to adapt code logic, add test cases, and documents